docs/validation/guides/validate-ODF-LSO.md

```diff
@@ -0,0 +1,44 @@
+# Validate Deployment of OpenShift Data Foundation (ODF) with LSO
+
+The steps below describe how to run acceptance tests manually for ODF (OpenShift Data Foundation) using LSO (Local Storage Operator).
+
+Prerequisites
+
+- OpenShift Cluster Installed in version 4.13 (or newer). Export the variable `CLUSTER_VERSION`. Example: `export CLUSTER_VERSION=4.13`
+- Cluster name exported in the variable `CLUSTER_NAME`
+- Cluster administrator credentials is saved under ${INSTALL_DIR}/auth, example:
+
+~~~bash
+$ tree ${INSTALL_DIR}
+└── auth
+    ├── kubeadmin-password
+    └── kubeconfig
+~~~
+
+Steps:
+
+1. Run a simple test:
+
+~~~bash
+podman run -v ${INSTALL_DIR}:/opt/cluster:Z quay.io/ocsci/ocs-ci-container:stable run-ci --cluster-path /opt/cluster --ocp-version ${CLUSTER_VERSION} --ocs-version ${CLUSTER_VERSION} --cluster-name ${CLUSTER_NAME} tests/manage/z_cluster/test_noobaa_xss_vulnerability.py --dev-mode
```

Comment:
[chatgpt] 
the ":Z" suffix is used to set the SELinux context for a volume mount.
By using the ":Z" option, you are telling Podman to relabel the volume with the correct SELinux context, allowing the container to access the content with the appropriate security policies enforced.


Can you add a comment if we run it from "non-Linux" OS we don't need to add ":Z"?